chips/earlgrey/src/plic.rs

```diff
@@ -42,7 +43,11 @@ pub unsafe fn clear_all_pending() {
 pub unsafe fn enable_all() {
     let plic: &PlicRegisters = &*PLIC_BASE;
     for enable in plic.enable.iter() {
-        enable.set(0xFFFF_FFFF);
+        // DANGER: For some reason USBDEV seems to be issuing
+        // unhandled CONNECT interrupts. So I have disabled them
+        // here. This should be changed back to 0xFFFFFFFF once
+        // this is resolved and before merging. -pal 8/26/20
+        enable.set(0x0000_00FF);
```

Comment:
This shouldn't be part of this PR

Reply:
Also in general this just covers up bugs, the real cause should be identified.

I thought this was a HW issue fixed in the more recent bit streams?

Reply:
Yes, I haven't gotten to the bottom of this yet -- it will be fixed in the eventual PR.

Reply:
#2100